client/src/components/Ticketing/ticketingmanager/AdminPurchase/AdminPurchase.tsx

```diff
@@ -478,14 +492,38 @@ const AdminPurchase = () => {
       headerName: '',
       width: 150,
       renderCell: (params) => (
-        <button
-          className='p-2 rounded-lg text-zinc-500 hover:text-red-600 hover:bg-red-100
-            focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-red-500'
-          onClick={() => removeRow(params.row.id)}
-          aria-label='Delete ticket'
-        >
-          <TrashCanIcon className='h-5 w-5' strokeWidth={2} />
-        </button>
+        !params.row.complimentary ? (
+          <button
+            className='bg-red-500 px-2 py-1 text-white rounded-xl hover:bg-red-600 disabled:opacity-40 m-2'
+            onClick={() => removeRow(params.row.id)}
+          >
+            Remove
+          </button>
+        ) : <select
+        className='w-full'
+        value={params.row.department || ''}
+        onChange={(e) => handleDepartmentChange(e, params.row)}
+      >
+        <option value=''>Select Department</option>
+        <option value='backHouse'>Back House</option>
+        <option value='frontHouse'>Front House</option>
+        <option value='inHouse'>In House</option>
+      </select>
+      ),
+    },
+    {
+      field: 'compAction',
+      headerName: '',
+      width: 150,
+      renderCell: (params) => (
+        params.row.complimentary ? (
+          <button
+            className='bg-red-500 px-2 py-1 text-white rounded-xl hover:bg-red-600 disabled:opacity-40 m-2'
+            onClick={() => removeRow(params.row.id)}
+          >
+            Remove
+          </button>
```

Comment:
Here as well

Reply:
Hey Henry, just changed it to the old trash icon and tested it and it works, ready to merge I think if you are ready.
![Screenshot_839](https://github.com/WonderTix/WonderTix/assets/97020914/d204e9e1-519d-4a1e-98f7-7297ba4e3715)